botchat.css

```diff
@@ -122,6 +116,11 @@
         position: relative;
     }
 
+    .wc-message-wrapper.carousel .wc-message {
```

Comment:
DOM and CSS selectors are separate syntaxes. In DOM, className are indeed separated by a space character, but in CSS selectors they have no space character. Another example would be an ID - in the DOM it's an attribute e.g. id="foo" but in CSS it's #foo